src/components/Presentation/FileUploadList.vue

```diff
@@ -0,0 +1,56 @@
+<template lang="pug">
```

Comment:
Why were the "Clear all" and "Start upload" buttons not moved into this component as well? Could you describe your use case and the rationale for breaking it up this way?

Reply:
I broke this out separately because I think smaller and more specialized components are more flexible.  The composition of these components implies a particular intent "I want to upload 1 or more files to a server, and I don't care what happens to the files when they're done uploading".

A dropzone is useful to me outside the context of an upload widget because I could read the files into client memory and immediately produce a visualization of the data. 

I'm actually using this in the metabolomics UI with dropzone and fileuploadlist, but without the clear and upload buttons because those controls are elsewhere on the page.  The upload widget is part of a larger form, and having those buttons in the widget as they are here is like putting the submit button in the middle of a form: I want to trigger the upload and form submit together as a single operation.   I want to hide the idea of "uploading files" from the user and present a feel of "processing data" without having to care that a server is involved.

Finally, those buttons are very low-effort to reproduce in another app.  I thought including them would reduce the flexibility to reuse the file list and add almost no value re: preventing code duplication.